### DISCUSSION TITLE
In an exported pdf file, why is the font of headers different from the font of main text?

### DISCUSSION THREAD

**Discussion by hao-hong** [quarto-dev/quarto-cli#4579] [Q&A]
In LaTex or RMarkdown, the default setting is a uniform font for the entire document. But in Quarto, the font of headers is different. Is there a special reason for this that I might learn to appreciate? 

Otherwise, what is the most convenient way to make the font uniform? I saw a few different ways to do this by googling; each seems a bit complicated than I expected.

**Comment by baptiste** [comment]
The answer depends on the class of LaTeX document used. For the default `scrartcl` class, the fonts can be switched back to the main font with the [somewhat non-obvious setting `egregdoesnotlikesansseriftitles`](https://tex.stackexchange.com/questions/6137/fonts-in-scrartcl)


`````
---
title: "test"
format: 
  pdf:
    mainfont: 'Zapfino'
    keep-tex: true
    classoption: egregdoesnotlikesansseriftitles
---

## Quarto

Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.

## Running Code

When you click the **Render** button a document will be generated that includes both content and the output of embedded code. You can embed code like this:

```{r}
1 + 1
```
`````

![Screenshot 2023-03-02 at 16 00 28](https://user-images.githubusercontent.com/18970/222319963-de469fa2-6b9b-4406-a5d6-a972d16fc078.png)
